### PR TITLE
remove sliced

### PR DESCRIPTION
**Pull request by Uzlopak**
Remove the package "sliced" in favor of native functions.

### CODE REVIEW

**Comment by Uzlopak** [comment]
Closing for now, as I am unsure if this has a perf impact.

**Comment by Uzlopak** [comment]
I checked mongoose sourcecode. It seems that the spread operator is not used usually, but everywhere Array.from. I can change the occurences of the spread operator to Array.from if it is desired.

File: lib/drivers/node-mongodb-native/collection.js [hunk 1]
```diff
@@ -173,7 +172,7 @@ function iter(i) {
     if (debug) {
       if (typeof debug === 'function') {
         debug.apply(_this,
-          [_this.name, i].concat(sliced(args, 0, args.length - 1)));
+          [_this.name, i].concat(args.slice(0, args.length - 1)));
```
[the remark this answers is not in the document]
Comment (vkarpov15) [reply lib/drivers/node-mongodb-native/collection.js:175]: Debug can be used in production. I'm sure some people use it for logging or perf. Shouldn't assume that certain features are never used in prod.

**Review by vkarpov15 — APPROVED** [review]
Nice work, thanks :+1: I like removing the extra dependency, especially if the benchmarks that justified using sliced are no longer accurate.

**Comment by AbdelrahmanHafez** [comment]
Nice work, thanks @Uzlopak 👍

**Pull request merged**